src/curated-corpus/helpers/prospects.ts

```diff
@@ -85,6 +85,7 @@ export const transformProspectToApprovedItem = (
     externalId: '',
     prospectId: prospect.prospectId,
     url: prospect.url,
+    hasTrustedDomain: true,
```

Comment:
👍 This is probably OK to hardcode to `true`, because the value is `hasTrustedDomain` is determined on the backend.